test/test_files/patcher_data/samus_returns/starter_preset.json

```diff
@@ -1,702 +1,3375 @@
+        },
+        "model": [
+          "item_supermissiletank"
+        ]
+      },
+      {
+        "pickup_type": "actor",
+        "caption": "Missile Tank acquired.\nMissile capacity increased by 3.",
+        "resources": [
+          [
+            {
+              "item_id": "ITEM_WEAPON_MISSILE_MAX",
+              "quantity": 3
+            }
+          ]
+        ],
+        "pickup_actor": {
+          "scenario": "s025_area2b",
+          "actor": "HP_Item_002"
+        },
+        "model": [
+          "item_missiletank"
+        ]
+      },
+      {
+        "pickup_type": "actor",
+        "caption": "Missile Tank acquired.\nMissile capacity increased by 3.",
+        "resources": [
+          [
+            {
+              "item_id": "ITEM_WEAPON_MISSILE_MAX",
+              "quantity": 3
+            }
+          ]
+        ],
+        "pickup_actor": {
+          "scenario": "s028_area2c",
+          "actor": "LE_Item_001"
+        },
+        "model": [
+          "item_missiletank"
+        ]
+      },
+      {
+        "pickup_type": "actor",
+        "caption": "Super Missile Tank acquired.\nSuper Missile capacity increased by 1.",
+        "resources": [
+          [
+            {
+              "item_id": "ITEM_WEAPON_SUPER_MISSILE_MAX",
+              "quantity": 1
+            }
+          ]
+        ],
+        "pickup_actor": {
+          "scenario": "s028_area2c",
+          "actor": "LE_Item_002"
+        },
+        "model": [
+          "item_supermissiletank"
+        ]
+      },
+      {
+        "pickup_type": "actor",
+        "caption": "Progressive Suit acquired.",
+        "resources": [
+          [
+            {
+              "item_id": "ITEM_VARIA_SUIT",
+              "quantity": 1
+            }
+          ],
+          [
+            {
+              "item_id": "ITEM_GRAVITY_SUIT",
+              "quantity": 1
+            }
+          ]
+        ],
+        "pickup_actor": {
+          "scenario": "s028_area2c",
+          "actor": "LE_Item_003"
+        },
+        "model": [
+          "powerup_variasuit",
+          "powerup_gravitysuit"
+        ]
+      },
+      {
+        "pickup_type": "actor",
+        "caption": "Aeion Tank acquired.\nAeion Gauge expanded.",
+        "resources": [
+          [
+            {
+              "item_id": "ITEM_MAX_SPECIAL_ENERGY",
+              "quantity": 50
+            }
+          ]
+        ],
+        "pickup_actor": {
+          "scenario": "s028_area2c",
+          "actor": "LE_Item_005"
+        },
+        "model": [
+          "item_senergytank"
+        ]
+      },
+      {
+        "pickup_type": "actor",
+        "caption": "Main Power Bomb acquired.",
+        "resources": [
+          [
+            {
+              "item_id": "ITEM_WEAPON_POWER_BOMB",
+              "quantity": 1
+            },
+            {
+              "item_id": "ITEM_WEAPON_POWER_BOMB_MAX",
+              "quantity": 5
+            }
+          ]
+        ],
```

Comment:
Yeah I still need to fix this, but I'd rather do it properly and all at once. Probably better for a separate pr since I also have to fix the db to match some patcher changes as it is, like adding in the baby data and adjusting metroid spawns, etc

Reply:
Ok, honestly, it's hard for me too know what the purpose of this pr is exactly if some things gets fixed and some not.
But as it is a non playable game atm, I don't care that much about doing other fixes in the future.

Reply:
TBH it's not organized the best. This is probably it for a while until the patcher really settles. It's kinda a waste to do tidbits when stuff changes all the time with the patcher. Future prs will be more focused.